Index: andv-front-end/.env.development
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>NODE_ENV=development\r\nVUE_APP_PREVIEW=true\r\nVUE_APP_API_BASE_URL=/api\r\n
===================================================================
diff --git a/andv-front-end/.env.development b/andv-front-end/.env.development
--- a/andv-front-end/.env.development	
+++ b/andv-front-end/.env.development	
@@ -1,3 +1,3 @@
 NODE_ENV=development
 VUE_APP_PREVIEW=true
-VUE_APP_API_BASE_URL=/api
+VUE_APP_API_BASE_URL=/api
\ No newline at end of file
Index: front-end/src/core/antdv_icon.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import {\r\n  GithubOutlined\r\n} from '@ant-design/icons-vue'\r\n\r\n// antd icon的请自行写入\r\n// https://2x.antdv.com/components/icon-cn/\r\nexport default {\r\n  install(app) {\r\n    app.component('GithubOutlined', GithubOutlined)\r\n\r\n  }\r\n}
===================================================================
diff --git a/front-end/src/core/antdv_icon.ts b/front-end/src/core/antdv_icon.ts
--- a/front-end/src/core/antdv_icon.ts	
+++ b/front-end/src/core/antdv_icon.ts	
@@ -9,4 +9,4 @@
     app.component('GithubOutlined', GithubOutlined)
 
   }
-}
\ No newline at end of file
+}
